landscape_config.yaml

```diff
@@ -0,0 +1,154 @@
+    domain_name: "<full domain name for cluster>"
+    # DNS provider (currently only route53 supported by these setup scripts)
+    # hosted zone for domain names and credentials (possibly the same ones
+    # as above)
+    hosted_zone_id: "<hosted zone id>"
+    access_key: "<access key for hosted zone>"
+    secret_key: "<secret key for hosted zone>"
+  master:
+    # Properties for master nodes.
+    count: 3
+    volume_size: 50
+  worker:
+    # Properties for worker nodes
+    count: 3
+    volume_size: 50
+
+misc:
+  variant_all:
+    # use htpasswd to create password entries
+    # example here: admin: ********* (htpasswd -bn admin "chosen password")
+    dashboard_creds: "<credentials here>"
+  variant_aws:
+  variant_openstack:
+    os_fip_pool_name: "<floating ip pool name>"
+
+# ---------------------------------------------------------------------------
+# Helm charts for gardener, dasboard, and identity deployments
+# ---------------------------------------------------------------------------
+
+charts:
+- name: gardener
+- name: identity
+  # identity is the identity provider for the dashboard
+  # this addon specifies who can login into the dashboard and how
+  # there are basically two options:
+  # - you specify accounts in this file: see "staticPasswords"
+  # - you use a dex connector to use OAuth authentication for the dashboard: see "connectors"
+  # You need at least one entry in one of the categories, otherwise the dashboard won't be accessible!
+  values:
+  # You can either provide the password in clear text
+  # in the "password" key or provide bcrypted passsword in
+  # the "hash" key. Bcrypted password can be created with
+  # htpasswd -bnBC 10 "" password | tr -d ':\n' | sed 's/$2y/$2a/'
+  staticPasswords:
+  # example 
+  - email: <email address>
+    hash: <bcrypted password>
+    username: <username>
+    userID: <UUID>
+  connectors:
+  # Example for a GitHub connector below
+  # You will need to configure your GitHub account accordingly
+  # see https://github.com/coreos/dex/blob/master/Documentation/connectors/github.md for further information on the GitHub connector
+  # and https://github.com/coreos/dex/tree/master/Documentation/connectors for more connectors
+  # uncomment and fill in to add the github connector
+  # (Github needs to be able to access the redirectURI for this to work!)
+#  - type: github
+#    id: github
+#    name: GitHub
+#    config:
+#      clientID: <github client id or environment variable containing it>
+#      clientSecret: <github client secret or environment variable containing it>
+#      # redirectURI: <identity URL (with https!)>/callback
+#      # identity is usually located at identity.ingress. + clusters.dns.domain_name
+#      redirectURI: <redirect URI>
+#      # Only users which are members of at least one organization can authenticate
+#      orgs:
+#      - name: <name of github organization>
+- name: certmanager
+  # Email address used for ACME registration
```

Comment:
link to https://en.wikipedia.org/wiki/Automated_Certificate_Management_Environment ?